### ISSUE TITLE
testing takes forever

### ISSUE THREAD

**Issue by Austernpilz** [OpenMS/OpenMS#6880]
https://github.com/OpenMS/OpenMS/blob/develop/src/tests/class_tests/openms/source/OpenPepXLLFAlgorithm_test.cpp
--- takes over 100 sec to test
https://github.com/OpenMS/OpenMS/blob/74f76cec5178eae044646b10b2a63c2705aa9e7d/src/tests/class_tests/openms/source/SimpleSVM_test.cpp#L4
--- takes over 100 sec to test
https://github.com/OpenMS/OpenMS/blob/74f76cec5178eae044646b10b2a63c2705aa9e7d/src/tests/class_tests/openms/source/OPXLHelper_test.cpp#L4
--- takes almost 200 sec to test

:((( I used 30 and 60 threads, but this is way too much waiting time for testing

**Comment by timosachsenberg** [comment]
Do you test in debug mode? These search engine tests are quite heavy if not optimized in release mode

**Comment by cbielow** [comment]
in my measurements, the times about 1/2 of the above, but still significant (in Release mode).
We do have `set(CMAKE_CXX_FLAGS_RELEASE "-O0")` for tests, but that does not change the timings a lot since it only affects the test.cpp itself, not the method calls into OpenMS.

**Comment by timosachsenberg** [comment]
is this on windows and maybe a lot of output produced?

**Comment by cbielow** [comment]
no, Linux

**Comment by timosachsenberg** [comment]
k thanks. 
the SImpleSVM_test is related to a too extensive cross-validation. I can provide a quick fix that can go into 3.0.1

**Comment by timosachsenberg** [comment]
I reduced it to about 30s https://github.com/OpenMS/OpenMS/pull/6951
Note that you can run tests in parallel mode so this one will complete before the last test was completed -> no slowdown here.

**Comment by timosachsenberg** [comment]
yeah this is true. I think the test is nice but for speed reasons we might need to make a linear test dataset with outliers and fit using a linear kernel

**Comment by timosachsenberg** [comment]
I did not observe any super slow tests anymore (in release mode)
